common/changes/@microsoft/rush/main_2022-09-20-15-16.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
```

Comment:
Are you sure we need to check in this file?

Reply:
Oops, this was committed while testing, forgot to remove it 😆